modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,219 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
 ======
 =====
 
+=== Access Neo4j using a Kubernetes Ingress
+
+The Neo4j Helm charts provide a helm chart that allows you to use a Kubernetes Ingress to access Neo4j running on port _:80_ or _:443_.
+The helm chart is called _neo4j/neo4j-reverse-proxy_ and is available on the _neo4j_ helm repository from version 5.12.0.
+For more information about Kubernetes Ingress, see the link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes official documentation -> Ingress].
+
+The helm chart creates a reverse proxy that is configured to route traffic to the Neo4j service URL using the `serviceName`, `namespace`, and `domain` values.
+For example, if the `serviceName` is _standalone-admin_, the `namespace` is _default_, and the `domain` is _cluster.local_, then the Neo4j service URL is _standalone-admin.default.svc.cluster.local_.
+
+image::reverse-proxy.svg[title="Reverse proxy flow diagram"]
+
+==== Configuration options
+
+To see all configurable options, run the following command:
+
+[source, shell]
+--
+helm show values neo4j/neo4j-reverse-proxy
+--
+
+[source, yaml]
+----
+# Default values for neo4j reverse proxy helm chart
+
+## @param nameOverride String to partially override common.names.fullname
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+fullnameOverride: ""
+
+# Parameters for reverse proxy
+reverseProxy:
+  image: "neo4j/neo4j-reverse-proxy"
+
+  # Name of the kubernetes service. This service should have the ports 7474 and 7687 open.
+  # This could be the admin service ex: "standalone-admin" or the loadbalancer service ex: "standalone" created via the neo4j helm chart
+  # serviceName , namespace , domain together will form the complete k8s service url. Ex: standalone-admin.default.svc.cluster.local
+  # When used against a cluster ensure the service being used is pointing to all the cluster instances.
+  # This could be the loadbalancer from neo4j helm chart or the headless service installed via neo4j-headless-service helm chart
+  serviceName: ""
+  # default is set to cluster.local
+  domain: "cluster.local"
+
+  # This assumes ingress-nginx controller or haproxy-ingress-controller is already installed in your kubernetes cluster.
+  # You can install ingress-nginx by following instructions on this link https://github.com/kubernetes/ingress-nginx/blob/main/docs/deploy/index.md#quick-start
+  # You can install haproxy-ingress by following instructions on this link https://haproxy-ingress.github.io/docs/getting-started/
+  ingress:
+    enabled: true
+    #default value is nginx. It can be either nginx or haproxy
+    className: nginx
+    annotations: {}
+#      "demo": "value"
+#      "demo2": "value2"
+    tls:
+      enabled: false
+      config: []
+#      - secretName: "demo2"
+#        hosts:
+#          - localhost
+----
+
+==== Install the Helm chart
```

Comment:
here we should mention in brackets (Using Nginx Ingress Controller) since this can work with haproxy as well but the example we are showcasing is for nginx ingress controller.